src/components/Chip/styles.css

```diff
@@ -0,0 +1,48 @@
+.chip-row {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: var(--tui-color-background-active-light);
+    padding: var(--tui-spacing-small) var(--tui-spacing-large);
+    border-radius: var(--tui-spacing-extra-large);
+    vertical-align: middle;
+
+    .label {
+        margin: 0 var(--tui-spacing-small);
+    };
+
+    .clickable {
+        cursor: pointer;
+    }
+
+    .disabled {
+        cursor: auto;
+    }
+
+    .icon {
+        align-self: center;
+    }
+
+    .disabled-label {
+        color:var(--tui-color-text-disabled-dark)
+    }
+
+    .action-component {
+        height: var(--tui-font-size-extra-large);
```

Comment:
Lets not do this, as action component has the possibility to contain more than one wider elements.
As Ankit suggested, make the label grow and don't allow the icon and action to shrink.